phpstan.neon.dist

```diff
@@ -24,7 +24,7 @@ parameters:
         - '~^Method Doctrine\\DBAL\\Driver\\SQLSrv\\SQLSrvConnection::errorCode\(\) should return string\|null but returns false\.\z~'
 
         # http://php.net/manual/en/pdo.sqlitecreatefunction.php
-        - '~^Call to an undefined method Doctrine\\DBAL\\Driver\\PDOConnection::sqliteCreateFunction\(\)\.\z~'
+        - '~^Call to an undefined method PDO::sqliteCreateFunction\(\)\.\z~'
```

Comment:
@Majkl578 while I understand the PHPStan's inability to use stubbed `PDO::SQLSRV_*` constants, what's the reason we should white-list this method? As long as `pdo_sqlite` is loaded on Travis, shouldn't PHPStan be able to see it via the extension reflection?

Reply:
> As long as pdo_sqlite is loaded

Exactly, as long as it is loaded. And pdo_sqlite is not direct dependency in composer.json, so you may not have it enabled locally. And polyfills can't add methods/constants to existing classes.
Maybe we should have two phpstan configs with specific ignores for Travis.

Reply:
This is als why I disabled `reportUnmatchedIgnoredErrors`, since different people/environments have different extensions.

Reply:
Gotcha. I thought this issue was reproducible despite whether `pdo_sqlite` was loaded or not.

The necessity to use `reportUnmatchedIgnoredErrors()` across the board and the fact that PHP extensions _can_ add members to existing classes (which is impossible in PHP) looks like a ground for PHPStan to implement this "extensibility" feature for stubs as well. I.e. it should initialize from the reflection extension and merge the stubbed symbols in. Does it sound like a reasonable feature request?